libs/cohere/pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "langchain-cohere"
-version = "0.1.1"
+version = "0.1.2-rc.1"
```

```suggestion
version = "0.1.2-rc1"
```

I think previous rc versions don't have a dot here - happy either way, though!